src/pages/People/People.tsx

```diff
@@ -0,0 +1,125 @@
+import { PeopleFilters } from '../../components/PeopleFilters/PeopleFilters';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable/PeopleTable';
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { useSearchParams } from 'react-router-dom';
+import { filter } from '../../function/filter';
+import { FILTER } from '../../types/filters';
+
+export const People = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState<boolean>(false);
```

Comment:
The `errorMessage` state is a boolean, but its name suggests it might hold an error message string. Consider renaming it to `hasError` for clarity.